### ISSUE TITLE
inputDialog on menu callback

### ISSUE THREAD

**Issue by Aaron-Downham** [overextended/ox_lib#77]
In the callback function on the lib menu, I'm trying to request an input from the player using lib.inputDialog however the input only appears for a split second before closing. Using v2.10.3 and if I print the input it's just an empty table; below is an example bit of code, just select the option in the menu and you should be able to reproduce the issue.

```
lib.registerMenu({
    id = "testMenu",
    title = "Test Menu",
    options = {
        {label = "Test"}
    }
}, function()
    local input = lib.inputDialog("Test", {"Input"})
    print(json.encode(input))
end)

RegisterCommand("test", function()
    lib.showMenu("testMenu")
end)
```

**Comment by LukeWasTakenn** [comment]
Seems to be happening because the input dialog is being set to visible and the confirm is being triggered from the enter key.

https://github.com/overextended/ox_lib/blob/4bd0e6936def637a80a0857fb603ea64cafec705/web/src/features/dialog/InputDialog.tsx#L67-L69

**Comment by LukeWasTakenn** [comment]
Should be resolved in commit ae25d109fc5127d8c84a91f98edd0c67382eeff2
